database/database.go

```diff
@@ -176,6 +182,62 @@ func (d *Database) PostPost(post *model.Post) (postid string, err error) {
 	return id, nil
 }
 
+func (d *Database) DeleteThread(threadId string) (err error) {
+	sqlStatement := `
+		DELETE FROM board.thread
+		WHERE Id = $1`
+	res, err := DB.Exec(sqlStatement, threadId)
+	if err != nil {
+		panic(err)
+	}
+	count, err := res.RowsAffected()
+	if err != nil {
+		panic(err)
+	}
+
+	if (count > 0) {
+		return
+	}
+
+	return errors.New("Couldn't find that thread")
+}
+
+func (d *Database) EditPost(editPost *model.Post) (err error) {
```

Comment:
SUGGESTION: Perhaps return the updated Post (all the way back). This isn't like 100% necessary, curious as to your thoughts on doing that.